EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/EventGenerator.java

```diff
@@ -143,7 +161,13 @@ private void tryToFillDependentAttributes(TypeTreeNode baseAttribute, Map<String
      * @param possibleValues a string containing the possible values which can be extracted by the "getRandom[attributeType]FromInput"-function
      * @param eventValues the map containing already set values will be altered by reference
      */
-    private void setRandomValueFromRangeForAttribute(TypeTreeNode attribute, String possibleValues, Map<String, Serializable> eventValues) {
+    private void setRandomValueFromRangeForAttribute(TypeTreeNode attribute, String proposedPossibleValues, Map<String, Serializable> eventValues) {
+        String possibleValues = proposedPossibleValues;
+
+        if(possibleValues == null || possibleValues.isEmpty()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') 'if' is not followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AWhitespace_Around_BP2016)